src/Storage/Field/Type/TaxonomyType.php

```diff
@@ -20,24 +20,6 @@ class TaxonomyType extends FieldTypeBase
 
     /**
      * Taxonomy fields allows queries on the parameters passed in.
-     * For example the following queries:
-     *     'pages', {'categories'=>'news'}
-     *     'pages', {'categories'=>'news || events'}.
-     *
-     * Because the search is actually on the join table, we replace the
-     * expression to filter the join side rather than on the main side.
-     *
-     * @param QueryInterface $query
-     * @param ClassMetadata  $metadata
-     */
-    public function query(QueryInterface $query, ClassMetadata $metadata)
-    {
-        $field = $this->mapping['fieldname'];
-
-        foreach ($query->getFilters() as $filter) {
-            if ($filter->getKey() == $field) {
-                // This gets the method name, one of andX() / orX() depending on type of expression
-                $method = strtolower($filter->getExpressionObject()->getType()) . 'X';
 
                 $newExpr = $query->getQueryBuilder()->expr()->$method();
                 foreach ($filter->getParameters() as $k => $v) {
```

Comment:
Not meant as code review, just flagging… But it looks like some :koala: copy/pasta